doc/template.html

```diff
@@ -67,5 +67,36 @@ <h3 class="date">$date$</h3>
 $for(include-after)$
 $include-after$
 $endfor$
+<script type="text/javascript">
+  (function() {
+    // write out a basic 'dropdown_link' reference element
+    var dropdown_link = document.createElement('a');
+    dropdown_link.className = 'dropdown_link';
+    dropdown_link.innerHTML = ' <b>+</b>'; // the space is for padding
+    dropdown_link.style.cursor = 'pointer';
+    document.querySelectorAll('nav ul li > ul > li').forEach(
+    function(element) {
+      var dropdown_link_copy = dropdown_link.cloneNode(true);
+      var list = element.querySelector('ul');
+      if (!list || list.querySelectorAll('li').length == 1)
```

Comment:
The reason I did this was to not hide items that had only one subelement. Either I can remove this entirely or keep it as this; there's no way other than this to detect whether there's `n` amount of elements.